<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Settings</class>
 <widget class="QDialog" name="Dialog_Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog_UserSettings {
  font-family: &quot;Courier New&quot;;
  background-color: #FFFFFF;
}
QGroupBox {
  font-family: &quot;Courier New&quot;;
  font-size: 13px;
  background-color: rgba(248, 249, 250, 1);
}
QLineEdit {
  padding: 4px 0;
  border: 1px solid #ced4da;
  border-radius: 8px;
  margin-left: 8px;
  padding-left: 4px;
  background-color: #FFFFFF;
  color:#212529;
}
QPlainTextEdit {
	background-color: #FFFFFF;
  color:#212529;
}
QLabel {
  font-family: &quot;Courier New&quot;;
  font-size: 13px;
  color: rgb(90, 93, 97);
}
QRadioButton {
  font-family: &quot;Courier New&quot;;
  font-size: 13px;
  color: #212529;
}
QComboBox {
  font-family: &quot;Courier New&quot;;
  font-size: 13px;
  color: #212529;
}
QPushButton {
  color: #212529;
}

QCheckBox{
  color: #212529;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <widget class="QRadioButton" name="proxy_radio">
            <property name="text">
             <string>Proxy</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="udd_radio">
            <property name="text">
             <string>User dir</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QRadioButton" name="re_template_radio">
            <property name="text">
             <string>RE Template</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="fields_container">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="udd_container" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="verticalLayout_">
               <item>
                <widget class="QWidget" name="udd_input_container" native="true">
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_11">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <property name="horizontalSpacing">
                   <number>4</number>
                  </property>
                  <item row="0" column="0">
                   <layout class="QVBoxLayout" name="verticalLayout_11">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="udd_label">
                      <property name="text">
                       <string>value</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="udd_input"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="udd_is_selected_checkbox">
                 <property name="text">
                  <string>Is selected</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="proxy_container" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QGridLayout" name="gridLayout_12">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="horizontalSpacing">
              <number>4</number>
             </property>
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="proxy_label">
                 <property name="text">
                  <string>value</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="proxy_input"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="re_template_container" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="re_template_container_layout">
               <property name="spacing">
                <number>4</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="part_layout">
                 <item>
                  <widget class="QRadioButton" name="title_radio">
                   <property name="text">
                    <string>Title</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="description_radio">
                   <property name="text">
                    <string>Description</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QComboBox" name="transaction_type_combobox"/>
               </item>
               <item>
                <widget class="QComboBox" name="categories_combobox"/>
               </item>
               <item>
                <widget class="QPlainTextEdit" name="value_plain_text"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>36</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>36</number>
            </property>
            <item>
             <widget class="QPushButton" name="create_new_btn">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="action_import_btn">
              <property name="text">
               <string>Import</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="action_export_btn">
              <property name="text">
               <string>Export</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
